app/models/disposable_domain.rb

```diff
@@ -1,2 +1,9 @@
 class DisposableDomain < ApplicationRecord
+  class << self
+    def is_disposable?(domain)
```

Comment:
style nit, I think the method would be more ruby-like if we dropped the `is_`
```suggestion
    def disposable?(domain)
```